packages/internal/cli/src/lib/runCommand.ts

```diff
@@ -0,0 +1,26 @@
+import { SpawnOptionsWithoutStdio, spawn } from 'node:child_process';
+
+export function runCommand(
+  command: string,
+  args: string[],
+  options?: SpawnOptionsWithoutStdio
+) {
+  return new Promise<void>((resolve, reject) => {
+    console.log([command, ...args].join(' '));
```

Comment:
It is intended to log command with all args?

Reply:
Yes, so you can see what is being executed